tests/data/datasets.py

```diff
@@ -4,9 +4,9 @@
 
 
 class SKLearnDataSets(Enum):
-    BOSTON = datasets.load_boston()
-    CANCER = datasets.load_breast_cancer()
-    DIABETES = datasets.load_diabetes()
-    LINNERUD = datasets.load_linnerud()
-    IRIS = datasets.load_iris()
-    WINE = datasets.load_wine()
+    BOSTON = datasets.load_boston().data
+    CANCER = datasets.load_breast_cancer().data
+    DIABETES = datasets.load_diabetes().data
+    LINNERUD = datasets.load_linnerud().data
+    IRIS = datasets.load_iris().data
+    WINE = datasets.load_wine().data
```

Comment:
Quite a weird issue. Python 3.5's Enum doesn't like sklearn's `Bunch` class. This workaround gets rid of random test failure (at least on my machine).